pkg/api/api_test.go

```diff
@@ -0,0 +1,30 @@
+package api
```

Comment:
Can we move this to api_test?

This makes it clearer that we are testing the package's public API

Reply:
should I move it in pkg/api_test or pkg/api/api_test?

Reply:
Don't move the file.

Just rename the package. `package api_test`

Reply:
And I though that the idea to have api_test.go in the same filte and package of api.go was to have access to all funcs!?

Reply:
You are only using public funcs.